docs/reference-guides/block-api/block-edit-save.md

```diff
@@ -188,7 +188,7 @@ When saving your block, you want to save the attributes in the same format speci
 
 ## Examples
 
-Here are a couple examples of using attributes, edit, and save all together. For a full working example, see the [Introducing Attributes and Editable Fields](/docs/how-to-guides/block-tutorial/introducing-attributes-and-editable-fields.md) section of the Block Tutorial.
```

Comment:
What happened to it?

Reply:
That section was removed here https://github.com/WordPress/gutenberg/pull/57120 but we missed the removal of this link. The idea according to that PR was that Building your first block tutorial would be a good replacement for that section. But in this context of a block API reference linking to the build your first block tutorial is not very useful.